.azure/app-cloud-e2e.yml

```diff
@@ -0,0 +1,140 @@
+# Python package
+# Create and test a Python package on multiple Python versions.
+# Add steps that analyze code, save the dist with the build record, publish to a PyPI-compatible index, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/python
+
+trigger:
+  tags:
+    include:
+      - '*'
+  branches:
+    include:
+      - "master"
+      - "release/*"
+      - "refs/tags/*"
+
+pr:
+  - "master"
+  - "release/*"
+
+variables:
+  # variables are automatically exported as environment variables so this will override pip's default cache dir
+  - name: pip_cache_dir
+    value: $(Pipeline.Workspace)/.pip
+
+jobs:
+  - job: App_cloud_e2e_testing
+    pool:
+      vmImage: 'ubuntu-latest'
+    timeoutInMinutes: "30"
+    cancelTimeoutInMinutes: "2"
+    strategy:
+      matrix:
+        'App: v0_app':
+          name: "v0_app"
```

Comment:
This PR should also update the job names for check group

Also the new name for this file

Reply:
> This PR should also update the job names for check group

done, note that with Azure we can name the group only no need to list all particular jobs unless we really need :)

> Also the new name for this file

what is wrong with it?

Reply:
> what is wrong with it?

This line: https://github.com/Lightning-AI/lightning/blob/9cf9bc582002a9fc3e0d3b890dc1aff9734cc870/.github/checkgroup.yml#L111 covered the previous job name. Now a new line needs to be added for `.azure/app-*`

Whenever we do changes to the CI filenames or the names of the jobs run per CI file, we need to keep in mind checkgroup

Reply:
updated, thx :)